src/main/java/java/lang/String.java

```diff
@@ -3884,6 +3884,46 @@ public String toString() {
         return result;
     }
 
+    /**
+     * Helper function for the {@code format} function.
+     * Serialize potential String.format argument to a String.
+     * The returned String is never null.
+     */
+    static String cproverFormatArgument(Object obj) {
+        if (obj == null) return "null";
+        if (obj instanceof String) return (String) obj;
```

Comment:
:pick: I realise I didn't review this PR properly. Why do you use that one line notation here? Does it come from OpenJDK? We usually use braces for conditionals.